libs/block_generator/block_generator.cpp

```diff
@@ -172,6 +172,9 @@ namespace {
 
     std::optional<std::string> check_json_config(const boost::json::value &json_value) {
         // Parse schema and check if the config is valid
+        if (!json_value.is_object()) {
+            return "Json value is empty";
```

Comment:
i try to check here if `jason_value` is empty (not initialized as member of `sequence_generator` for example), and not try to check scheme i so.
May be any better way for check if `json_value` is not initialized?

Reply:
I think you are not obliged to to keep `json_value` as a member and initialize it in constructor, it can be a parameter of `generate_block`. But if you want to have it this way, you could introduce additional flag member to keep the information about the initialization.